backend/src/main/java/ai/giskard/service/AnalyticsCollectorService.java

```diff
@@ -0,0 +1,104 @@
+package ai.giskard.service;
+
+import ai.giskard.config.GiskardConstants;
+import ai.giskard.service.ee.LicenseService;
+import com.mixpanel.mixpanelapi.MessageBuilder;
+import com.mixpanel.mixpanelapi.MixpanelAPI;
+import lombok.RequiredArgsConstructor;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+
+@Service
+@RequiredArgsConstructor
+public class AnalyticsCollectorService {
+    private static final Logger log = LoggerFactory.getLogger(AnalyticsCollectorService.class);
+
+    private static final String DEV_MP_PROJECT_KEY = "4cca5fabca54f6df41ea500e33076c99";
+    private static final String PROD_MP_PROJECT_KEY = "2c3efacc6c26ffb991a782b476b8c620";
```

Comment:
It's better to have only one variable that is injected using `@Value` from the profile